src/transformers/modeling_utils.py

```diff
@@ -2299,6 +2323,10 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         with ContextManagers(init_contexts):
             model = cls(config, *model_args, **model_kwargs)
 
+        if use_keep_in_fp32_modules:
+            low_cpu_mem_usage = True
+        keep_in_fp32_modules = model._keep_in_fp32_modules
```

Comment:
Let's set it to `[]` here if it's not `None`, so that we don't have to check again layer on.

Reply:
This should be addressed in [cb89c42](https://github.com/huggingface/transformers/pull/20683/commits/cb89c42fd2cd05aacb56c15462869c229faaf3fa)